experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/utils/IOScope.kt

```diff
@@ -0,0 +1,6 @@
+package example.imageviewer.utils
+
+import kotlinx.coroutines.CoroutineDispatcher
+
+expect
+val ioDispatcher: CoroutineDispatcher
```

Comment:
```suggestion
expect val ioDispatcher: CoroutineDispatcher
```
Accindentally a "return" character here

Reply:
Thanks!